garnbarn_api/views.py

```diff
@@ -16,12 +18,14 @@ class AssignmentViewset(viewsets.ModelViewSet):
     serializer_class = AssignmentSerializer
 
     def get_queryset(self):
+        user_data = self.request.user.get_json_data()["uid"]
```

Comment:
Maybe this can be minimized by `self.request.user.uid`

Reply:
Done in commit bc03e83.